src/rdb.c

```diff
@@ -1239,6 +1240,25 @@ int rdbSaveRio(rio *rdb, int *error, int rdbflags, rdbSaveInfo *rsi) {
     if (rdbSaveInfoAuxFields(rdb,rdbflags,rsi) == -1) goto werr;
     if (rdbSaveModulesAux(rdb, REDISMODULE_AUX_BEFORE_RDB) == -1) goto werr;
 
+    /* save functions */
+    dict* functions = functionsGet();
+    dictIterator *iter = dictGetIterator(functions);
```

Comment:
You need to make up your mind about where the pointer *s go :)

Reply:
At first I did everything like this one (pointer close to the type). But then I saw that the conversion in Redis is that the pointer is next to the variable name and I changed. Must a missed some. Will fix and re-scan the code, thanks.